images/diagnostics/Dockerfile

```diff
@@ -0,0 +1,9 @@
+#
+# OpenShift diagnostics image
+# Used by network diagnostics (oadm diagnostics NetworkCheck)
+#
+# The standard name for this image is openshift/diagnostics-deployer
+
+FROM docker.io/busybox
```

Comment:
openshift/origin-base image doesn't help in anyway, we don't need openshift related binaries here. We depend on openshift running on the node, so any small image with shell support is preferred.